NEWS.md

```diff
@@ -7,6 +7,16 @@ New language features
 Language changes
 ----------------
 
+ - When methods are replaced with exactly equivalent ones, the old method is no
+   longer deleted implicitly simultaneously, although the new method does take
+   priority and become morespecific than the old method. Thus if the new method
+   is deleted later, the old method will resume operating. This can be useful
+   to mocking frameworks (such as has been seen in SparseArrays, Pluto,
```

Comment:
```suggestion
   to mocking frameworks (such as SparseArrays, Pluto,
```

SparseArrays is a mocking framework?

Reply:
It has one, yes. It uses it to disable scalar indexing for testing.